app/javascript/components/schedule-form/helper.js

```diff
@@ -0,0 +1,82 @@
+// eslint-disable-next-line camelcase
+const customAttribute = (custom_attributes, name) => {
+  const attr = _.find(custom_attributes, { name });
+  return (attr && attr.serialized_value) || null;
+};
+
+export const getFormInitialValues = (recordId, name, description, setState) => {
+  debugger;
+  const timezoneOptions = [];
+  API.get('/api').then(({ timezones }) => {
+    timezones.forEach((timezone) => {
+      timezoneOptions.push({ value: timezone.name, label: timezone.description });
+    });
+    debugger;
+    // return timezoneOptions;
+    const initialValues = {
+      name,
+      description,
+    };
+    setState({
+      initialValues,
+      isLoading: false,
+      timezoneOptions,
+    });
+  });
+};
+
+export const getInitialValues = (recordId, name, description, setState) => {
+  const initialValues = {
+    name,
+    description,
+  };
+  setState({
+    initialValues,
+    isLoading: false,
+  });
+};
+
+export const getSubmitData = (values) => {
+  let customIdentifier = '';
+  let description = '';
+  let parentResource = null;
+  const childResources = [];
+
+  if (values.custom_1) {
+    customIdentifier = values.custom_1;
+  }
+
+  if (values.description) {
+    description = values.description;
+  }
+
+  if (values.parent_vm) {
+    const [id, isTemplate] = values.parent_vm.split(',');
+    if (isTemplate === 'true') {
+      parentResource = { href: `/api/templates/${id}` };
+    } else {
+      parentResource = { href: `/api/vms/${id}` };
+    }
```

Comment:
we can use like this

`return (isTemplate === 'true') ? { href: `/api/templates/${id}` } : { href: `/api/vms/${id}` };`